src/components/ChatPopUp.tsx

```diff
@@ -93,6 +93,10 @@ export interface ChatPopUpProps
    * Whether to show the initial message popup when the panel is hidden on load.
    * Defaults to false.
    */
+
```

Comment:
the comment above is for `showInitialMessagePopUp` prop, you should move the added botId field elsewhere